docs/examples/userguide/wrapping_CPlusPlus/iterate.pyx

```diff
@@ -0,0 +1,15 @@
+# distutils: language = c++
+
+from libcpp.vector cimport vector
+
+def main():
+    cdef vector[int] v
+    cdef int i, value
+
+    for i in range(10):
+        v.push_back(i)

```

Comment:
I think this for-loop is distracting. Just use a Python list assignment to the vector instead.